src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/AdminService.kt

```diff
@@ -0,0 +1,66 @@
+package uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.service
+
+import jakarta.transaction.Transactional
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.jpa.entity.CaseNoteRetryEntity
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.jpa.repository.CaseNoteRetryRepository
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.service.external.ResettlementPassportDeliusApiService
+import java.time.LocalDateTime
+
+@Service
+class AdminService(
+  private val caseNoteRetryRepository: CaseNoteRetryRepository,
+  private val caseNotesService: CaseNotesService,
+  private val resettlementPassportDeliusApiService: ResettlementPassportDeliusApiService,
+  private val metricsService: MetricsService,
+) {
+  companion object {
+    val log: Logger = LoggerFactory.getLogger(this::class.java)
+    const val MAX_RETRIES = 10
+  }
+
+  @Transactional
```

Comment:
I think it would be better to create a transaction for each separate retry - it would be bad if we sent some case notes, then failed and rolled back the status of which ones we had done